src/nrnpython/nrnpython.cpp

```diff
@@ -182,8 +203,13 @@ extern "C" int nrnpython_start(int b) {
         copy_argv_wcargv(nrn_global_argc, nrn_global_argv);
         PySys_SetArgv(nrn_global_argc, wcargv);
         nrnpy_augment_path();
-#if !defined(MINGW)
-        // cannot get this to avoid crashing with MINGW
+
+        // Used to crash with MINGW when assocated with a python gui thread e.g
+        // from neuron import h, gui
+        // g = h.Graph()
+        // del g
+        // Also, NEURONMainMenu/File/Quit did not work. The solution to both
+        // seems to be to just avoid gui threads if MINGW and launched nrniv
         PyOS_ReadlineFunctionPointer = nrnpython_getline;
 #endif
```

Comment:
Shouldn't the `#endif` go as well?

Reply:
Good catch. I removed the wrong #endif. See line 213 below.